package.json

```diff
@@ -19,6 +19,7 @@
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz",
+    "react-native-gesture-handler": "~1.10.2",
```

Comment:
It is better to fix the dependency version.

Reply:
I disagree :P The wave is a good compromise between a fixed version and a version patched (we usually want to benefit from the patches). The `yarn.lock` ensures the dependencies are fixed.